editor/js/Viewport.Info.js

```diff
@@ -45,28 +45,39 @@ Viewport.Info = function ( editor ) {
 				if ( object instanceof THREE.Mesh ) {
 
 					var geometry = object.geometry;
+					var indexCount;
 
 					if ( geometry instanceof THREE.Geometry ) {
 
 						vertices += geometry.vertices.length;
-						triangles += geometry.faces.length;
+						indexCount = geometry.faces.length * 3;
 
 					} else if ( geometry instanceof THREE.BufferGeometry ) {
 
 						vertices += geometry.attributes.position.count;
 
 						if ( geometry.index !== null ) {
 
-							triangles += geometry.index.count / 3;
+							indexCount = geometry.index.count;
 
 						} else {
 
-							triangles += geometry.attributes.position.count / 3;
+							indexCount = geometry.attributes.position.count;
 
 						}
 
 					}
 
+					if ( object.drawMode === THREE.TrianglesDrawMode ) {
```

Comment:
It seems the PR assumes that the editor is always used with `WebGLRenderer`. However, you can configure a different renderer in the project settings (`SVGRenderer`, `SoftwareRenderer` or `RaytracingRenderer`). It's probably better to retrieve the `triangles` property from `renderer.info` and not compute it on-the-fly. However, `SoftwareRenderer` and  `RaytracingRenderer` do not have an `info` property. To clean things up, it's necessary that all renderers have the same `info` interface. In the next step, it can be used in the editor.